sillsdev/web-languageforge
# Remove old user-management code

Follow up to #1556

## Description

While migrating to Playwright we noticed this old user-management page that's not in use anymore. Boom 💥.
In the process I found a handleful of other things that aren't used/referenced.

[This change](https://github.com/sillsdev/web-languageforge/compare/chore/remove-old-usermanagement-code?expand=1#diff-580d19f09eb3b266bc67b13adf55aa3c3a52955d4f73b39a67c6e4992a5d23d2) was made, because of a console error when clicking on a user-name here:
![image](https://user-images.githubusercontent.com/12587509/209139607-fc41a97b-f886-4994-b638-c8b9eb10ca69.png)
I think this is a remnant of removing jQuery, but I'm a bit confused. Anyway, it seems fine now. 🤷

### Type of Change

- Cleanup

## Checklist


## How to test

Smoke test to make sure everything's still working and I didn't remove something I shouldn't have. Perhaps particularly using the "share project" features, which are related to the majority of the code I removed.

## qa.languageforge.org testing

Testers should add his/her findings to end of the PR in a comment and include screenshots, files, etc that are beneficial.
